reference_guide/project_model/sdk.md

```diff
@@ -51,9 +51,13 @@ To make your SDK settings persistant, you should override `setupSdkPaths` and sa
 public boolean setupSdkPaths(@NotNull Sdk sdk, @NotNull SdkModel sdkModel) {
     SdkModificator modificator = sdk.getSdkModificator();
     modificator.setVersionString(getVersionString(sdk));
-    modificator.commitChanges();
+    modificator.commitChanges(); // save
     return true;
 }
 ```
 
-To let user select an SDK, see [ProjectJdksEditor](https://upsource.jetbrains.com/idea-ce/file/idea-ce-8c9022ae739b82b2ee8f3355da98b9bbce2cb915/java/idea-ui/src/com/intellij/openapi/projectRoots/ui/ProjectJdksEditor.java).
+To let user select an SDK, see [ProjectJdksEditor](upsource:///java/idea-ui/src/com/intellij/openapi/projectRoots/ui/ProjectJdksEditor.java).
+
+However, it is not recommended to use "SDK" in non-IDEA IDEs. "SDK" is IntelliJ-specific (as well as `ProjectJdksEditor` mentioned above) and doesn't work in PyCharm, RubyMine, etc.
```

Comment:
Can you please remove this paragraph? It's incorrect: the SDK infrastructure in the project model is fully supported in all IDEs, and is actively used at least in PyCharm and RubyMine. Only the `ProjectJdksEditor` is specific to IntelliJ IDEA.

Reply:
Oh..... Ok, let me change this